pkg/kwokctl/cmd/create/cluster/cluster.go

```diff
@@ -127,14 +128,35 @@ func runE(ctx context.Context, flags *flagpole) error {
 		defer cancel()
 	}
 
-	buildRuntime, ok := runtime.DefaultRegistry.Get(flags.Options.Runtime)
-	if !ok {
-		return fmt.Errorf("runtime %q not found", flags.Options.Runtime)
+	var runtimeCandidate []string
+	if flags.Options.Runtime != "" {
+		runtimeCandidate = []string{flags.Options.Runtime}
+	} else {
+		runtimeCandidate = []string{
```

Comment:
Add this to the KwokctlConfiguration to configure

Reply:
Kwok load config in runtime, we may need some refactoring to load the config before automatically selecting a runtime.

Reply:
I think no need to refactor the configuration loading, 

Also, do the runtime detection here based on the configuration.

Reply:
fixed this time, PTAL @wzshiming